dynamodb/app_storage.go

```diff
@@ -261,6 +261,15 @@ func (as *AppStorage) DeleteApp(id string) error {
 	return err
 }
 
+// TestDatabaseConnection checks whether we can fetch the first document in the applications table.
+func (as *AppStorage) TestDatabaseConnection() error {
+	_, err := as.db.C.Scan(&dynamodb.ScanInput{
+		TableName: aws.String(AppsTable),
+		Limit:     aws.Int64(1),
+	})
+	return err
```

Comment:
how about not found error? it shode be considered as successful result as well

Reply:
DynamoDB API does not return notfound errors on Scan: https://docs.aws.amazon.com/sdk-for-go/api/service/dynamodb/#DynamoDB.Scan